pkg/splunk/common/urls.go

```diff
@@ -98,6 +98,12 @@ const (
 	//LicenseManager = "license-master"
 	LicenseManager = "license-master"
 
+	// LicenseManagerRole = "splunk_license_master"
+	LicenseManagerRole = "splunk_license_master"
+
+	// LicenseManagerURL = "SPLUNK_LICENSE_MASTER_URL"
+	LicenseManagerURL = "SPLUNK_LICENSE_MASTER_URL"
+
```

Comment:
Should we create a followup issue for this once they are fixed within `splunk-ansible`?

Reply:
Absolutely. We're also reviewing this exception list to remove all the APIs that now ES supports with `/manager/`. But the complete list will come with the ClusterManager CRD (next PR).